applications/sam/sam_ls.py

```diff
@@ -158,8 +158,7 @@ def _printItem(self, item, args, level):
 
         # sam-ls --relative-path
         if args.relativePath:
-            filePath += (item.getFolder() if item.getFolder()[0] != '/' else '.')
-            filePath += ('/' if filePath[-1] != '/' else '')
+            filePath += ('.' if os.path.isabs(item.getFolder()) else item.getFolder())
```

Comment:
I don't understand what you are doing here. It should simply be:
`filePath += item.getFolder()`
getFolder() contains the path as given by the user. So if it's absolute, it will stay absolute and if it's relative it will stay relative.

Do I missed something?